tests/models/autoencoders/test_models_autoencoder_oobleck.py

```diff
@@ -132,6 +132,15 @@ def test_layerwise_casting_inference(self):
     def test_layerwise_casting_memory(self):
         pass
 
+    @unittest.skip(
```

Comment:
For these exceptions, I would add a `_supports_group_offloading=True` to ModelMixin and set to False on unsupported models.